src/lib/chat/matrix-client.ts

```diff
@@ -338,7 +338,8 @@ export class MatrixClient implements IChatClient {
       // as zOS considers any conversation to have ever had more than 2 people to not be 1 on 1
       isOneOnOne: room.getMembers().length === 2,
       otherMembers: otherMembers,
-      lastMessage: lastMessage,
+      lastMessage: null,
```

Comment:
also want to understand why we're not using the lastMessage now. Since we're fetching all messages, and this could be set as the last message from that list..?